certified-connectors/Converter by Power2Apps/apiDefinition.swagger.json

```diff
@@ -3,676 +3,2194 @@
+          "text/plain",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "JSON",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "JSON",
+            "description": "JSON"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
+        "summary": "Convert JSON to CSV (deprecated)",
+        "description": "Convert JSON to CSV"
+      }
+    },
+    "/ConvertJsonToCsv": {
+      "post": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "ConvertJsonToCsv",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "jsonData",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/JsonData"
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "JSON",
+            "description": "JSON"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CsvResponse"
+            }
+          }
+        },
+        "x-ms-visibility": "important",
+        "summary": "Convert JSON to CSV",
+        "description": "Convert JSON to CSV"
+      }
+    },
+    "/ConvertCsvdataToJson": {
+      "post": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "ConvertCsvdataToJson",
+        "consumes": [
+          "text/plain",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "CSV data",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "CSV",
+            "description": "CSV data"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
```

Comment:
@stsiek The operation has been hidden. This change breaks the designer for existing flows.

Reply:
we would like to set this action to deprecated, so it is set to "internal". we had understood that it can still be used for existing flows but is no longer visible when adding new actions. We also tested it and it works fine.